### ISSUE TITLE
models 'mobilenetv2_w1_bsconvs' and ''mobilenetv2_w1' are identical

### ISSUE THREAD

**Issue by chuong98** [zeiss-microscopy/BSConv#5]
I print out the two models, and don't see any difference. Reproduce:
```bash
import bsconv.pytorch
model1= bsconv.pytorch.get_model('mobilenetv2_w1_bsconvs',num_classes=100)
print(model1)
```
and 
```bash
import bsconv.pytorch
model2= bsconv.pytorch.get_model('mobilenetv2_w1',num_classes=100)
print(model2)
```

**Comment by sunzhe09** [comment]
me too

**Comment by zeiss-mamthor** [comment]
Hi,
as already discussed in issue #4 ([bullet point 1](https://github.com/zeiss-microscopy/BSConv/issues/4#issuecomment-666404746)) and described in the paper (Sect. 4.2), we retain the layer arrangement of the original model.
However, even though for this specific case the layers are the same, the BSConvS variant uses the additional regularization loss which causes the improved results. **This is not shown when printing the model.**

**Comment by sunzhe09** [comment]
@zeiss-mamthor I want to replace the MobileNetV2  bottleneck ，should I replace DepthwiseConv with BSConvS ?or just replace  invert_residual with BSConvS？I ‘m a little  confused

**Comment by zeiss-mamthor** [comment]
Following the implementation in the paper, the entire linear bottleneck block (not only the DW conv) is replaced by a special form of a BSConvS module, where the layer cascade of PW-PW-DW is shifted to match the original MobileNet design of PW-DW-PW. The reason is that in the MobileNet paper it was shown that residuals in the bottleneck provide a better accuracy/FLOP ratio compared to residuals outside of the bottleneck (as implemented in ResNets). We keep these findings for our models, i.e., converting a MobileNetV2 into its BSConvS variant consists of adding the regularization loss to the last PW conv of each bottleneck block while keeping the overall layer design.
You can simply use the [transformer for MobileNetV2/3](https://github.com/zeiss-microscopy/BSConv/blob/203fd9d2c604894ce540ffc6d3c66e541110cf39/bsconv/pytorch/mobilenet.py#L429) which adds the required function to the corresponding modules or just load an already converted model:

```python
import bsconv.pytorch
model = bsconv.pytorch.get_model('cifar_mobilenetv2_w1_bsconvs', num_classes=100)
```

Btw, do not forget to add the regularization loss during training:

```python
output = model(images)
loss = criterion(output, target)

# THIS LINE MUST BE ADDED, everything else remains unchanged
loss += model.reg_loss(alpha=0.1)
```
